### DISCUSSION TITLE
Bounty Prices

### DISCUSSION THREAD

**Discussion by Titan-Node** [livepeer/bounties#42] [Feedback]
Just a follow up conversation around the LPT reward price given to people who complete a bounty.

I think it is imperative that we offer relatively high rewards for people completing bounties.

A number of benefits occur:
1. People are motivated to try their first bounty and to start digging into the code, especially if they are new to Livepeer
2. It builds enthusiasm around Livepeer and introduces a curiosity of "maybe I could get involved full/part time" since the rewards here are so lucrative.
3. It shortens the time frame for how quickly a bounty will get completed by creating an opportunity for competition.

When considering an LPT amount, I think we could offer 1.5X to 2X the current amounts.

And in the future, with the success of the program, we could possibly request a separate amount from the Treasury dedicated to Bounties.

**Comment by rickstaa** [comment]
Hey @Titan-Node,

Thanks for your feedback. I agree that offering more generous rewards for these bounties makes sense, as it could attract high-quality Open Source contributors who might even make this a full-time profession. To achieve this, we need to consider the lack of security these contributors have compared to working at a company and the volatility of payments in LPT.

Currently, since the money for the AI Bounties (5000 LPT) was allocated by the community after a treasury vote, I aimed for what I believe is fair compensation. This was based on my estimation of the time a mid-level developer would need for the task, adding a 40% safety margin. I then looked at the average rate for a software engineering consultant in the US and converted that amount to the one-month average price of Livepeer (see [this Discord post](https://discord.com/channels/423160867534929930/1187806216185974934/1269239098418266142)).

However, if the community is open to adding a bonus amount, such as an extra 50%, I'm happy to factor that into the calculation 👍🏻.

I'm curious to hear the thoughts on this from the rest of the community.

**Comment by rickstaa** [comment]
@Titan-Node I turned this in a poll so that people can vote on it 👍🏻. Please checkout https://github.com/livepeer/bounties/discussions/43#discussion-7017108.

**Comment by rickstaa** [comment]
Replaced by https://github.com/livepeer/bounties/discussions/43. Please vote and give your 2 cents 🙏🏻.
